aixplain/modules/model/index_model.py

```diff
@@ -50,14 +52,60 @@ def __init__(
         )
         self.url = config.MODELS_RUN_URL
         self.backend_url = config.BACKEND_URL
+        self.embedding_model = embedding_model
```

Comment:
Where is this used? We can add it to the dictionary representation of `IndexModel`.

Reply:
This is used during indexing and is a required parameter to create an index. However, backend does not return it yet when getting the index model from the database. Backend team is working on it.

You mean added as a representation [here](https://github.com/aixplain/aiXplain/blob/main/aixplain/modules/model/__init__.py#L104)? I am not sure, since this is not an input parameter itself.